### ISSUE TITLE
Ticket forces orientation to portrait even if not supported

### ISSUE THREAD

**Issue by acegreen** [tradingticket/TradeItIosTicketSDKLib#51]
When I call the sheet on my app which only support landscape for iPad, it appears portrait. This causes the app to crash. didn't it use to appear as form sheet?

**Comment by acegreen** [comment]
Potentially fixed in V2.2.9

**Comment by danielvaughn** [comment]
Sorry, this latest version fixed a different issue. We'll have this one fixed soon.

**Comment by acegreen** [comment]
Ah ok, it looked fixed as it was working on simulator. Just tried it on device, and yes still an issue.

**Comment by acegreen** [comment]
@danielvaughn any update on this?

**Comment by danielvaughn** [comment]
@acegreen yes, it's fixed now but there are a couple of other things we need to include in the release that are still being hashed out. I'm hoping to get it out either this evening or tomorrow morning. Thanks for your patience.

**Comment by acegreen** [comment]
@danielvaughn any chance this will be an amazing update that includes issue #41

**Comment by danielvaughn** [comment]
Unfortunately #41 is a much larger project that we have to place further down our roadmap

**Comment by acegreen** [comment]
Yea I suspected, one can only dream though. Can we prioritize this though? I would like to take a vote with other developers and your team as I think holding out just adds more debt. Also I suspect that because this is a bigger task, the team may be subconsciously putting it on the back burner.

**Comment by danielvaughn** [comment]
@acegreen as of the latest release, version 2.3.0, this issue is now fixed. Thank you for your patience.

**Comment by acegreen** [comment]
Still forces orientation to portrait though it doesn't crash.

**Comment by danielvaughn** [comment]
For the time being, our SDK only supports portrait orientation. We're considering supporting landscape but it's further down the roadmap for us.

**Comment by acegreen** [comment]
But I recall it previously working? Did we eliminate that? To be honest, this is likely to be the final straw for me. I have already attempted to integrate this library several times but always ran into issues. As already mentioned, I think the current implementation is causing a lot of issues for which the team seems to be only finding workarounds, rather than implementing it in a clean way from the get-go. Separation of UI from API is critical and native implementation of UI elements and behaviour is a must. Current implementation feels like a port rather than a native setup. Please reconsider your approach going forward before there is too much debt. For now I'll be dropping this from my build. 

I have been very vocal because I was super excited about this library and having it within my app. Unfortunately it just doesn't work and the direction the team is taking isn't going to address this.
